v0.25.3 release preparation

cpu opened the pull request:
## Description

This branch targets the `rel-0.25` branch, which was branched from 22702f5e206604890349983f9c334dbc244d6f18, the last commit on `main` that's semver compatible with the `0.25.2` release cut from 4730449082b71d0a4f704b9180711e4d03995fc2. It backports the CCADB trust root updates from https://github.com/rustls/webpki-roots/pull/52 to prepare v0.25.3

## Proposed Release Notes

New trust anchors:
* [CommScope Public Trust RSA Root-01](https://bugzilla.mozilla.org/show_bug.cgi?id=1763177)
* [CommScope Public Trust RSA Root-02](https://bugzilla.mozilla.org/show_bug.cgi?id=1763177)
* [CommScope Public Trust ECC Root-01](https://bugzilla.mozilla.org/show_bug.cgi?id=1763177)
* [CommScope Public Trust ECC Root-02](https://bugzilla.mozilla.org/show_bug.cgi?id=1763177)
* [TrustAsia Global Root CA G4](https://bugzilla.mozilla.org/show_bug.cgi?id=1688854)
* [TrustAsia Global Root CA G3](https://bugzilla.mozilla.org/show_bug.cgi?id=1688854)

Removed trust anchors:
* Duplicated [Autoridad de Certificacion Firmaprofesional CIF A62634068](https://bugzilla.mozilla.org/show_bug.cgi?id=1842935)

djc commented on src/lib.rs:
```diff
@@ -50,6 +50,50 @@ pub const TLS_SERVER_ROOTS: &[TrustAnchor] = &[
     name_constraints: None
   },
 
+  /*
```
Is there a reason for not cherry-picking my commit here? Feels like that might help establishing that the same changes were made in both branches?

cpu replied:
No particular reason. I'll adjust :+1:

cpu replied:
Adjusted to use a cherry-pick. I had to manually fix the usage of pki-types + webpki API changes, so it's not a 1:1 with the commit from your `main` but there's no avoiding that.

djc reviewed (commented):
Why not branch from 22702f5e206604890349983f9c334dbc244d6f18? Seems a bit more clear in terms of the commit history.

cpu commented:
> Why not branch from https://github.com/rustls/webpki-roots/commit/22702f5e206604890349983f9c334dbc244d6f18? Seems a bit more clear in terms of the commit history.

As in start the `rel-0.25` branch from 22702f5e206604890349983f9c334dbc244d6f18? In the past when I've made the `rel-xxx` branches I've always branched from whatever commit was used for the most recent release in that stream, usually the one bumping the `Cargo.toml` version. It felt like the point releases should always pick up from where the last release left off :thinking:

djc commented:
> As in start the `rel-0.25` branch from [22702f5](https://github.com/rustls/webpki-roots/commit/22702f5e206604890349983f9c334dbc244d6f18)? In the past when I've made the `rel-xxx` branches I've always branched from whatever commit was used for the most recent release in that stream, usually the one bumping the `Cargo.toml` version. It felt like the point releases should always pick up from where the last release left off 🤔

It seems fair game to me to pick a later commit from the main branch that is still compatible with the 0.25.x range. If we make this clear in the PR description (maybe including mentioning the first PR that introduced incompatible changes), I think that would be a cleaner way of maintaining the commit history.

cpu commented:
I think that's reasonable for this repo where it's very clear what commit is the delineating point between compatible/incompatible. I will adjust :+1:

cpu commented:
* Force pushed an updated `rel-0.25` branched from https://github.com/rustls/webpki-roots/commit/22702f5e206604890349983f9c334dbc244d6f18
* Updated PR desc
* Added a `Cargo.toml` version bump for 0.25.3.

cpu commented:
* Pushed a [v0.25.3 tag](https://github.com/rustls/webpki-roots/releases/tag/v%2F0.25.3)
* Published webpki-roots v0.25.3 at registry `crates-io`
* Created a [GitHub release](https://github.com/rustls/webpki-roots/releases/tag/v%2F0.25.3)

I also noticed we were missing tags for 0.25.2, 0.25.1, and 0.25.0. I retroactively pushed all of those tags to GitHub (_but haven't created GitHub releases for all of the historic tagged Cargo releases_)
